lib/rack/session/redis.rb

```diff
@@ -14,7 +14,9 @@ def initialize(app, options = {})
         super
 
         @mutex = Mutex.new
-        @pool = ::Redis::Store::Factory.create @default_options[:redis_server]
+        @pool = @default_options.has_key?(:redis_store) ?
```

Comment:
But we can't evaluate second argument of this call unless `@default_options[:redis_server]` exists...